rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/10_synonyms_put.yml

```diff
@@ -5,9 +5,6 @@ setup:
 
 ---
 "Create synonyms with no ID creates ID automatically":
-  - skip:
```

Comment:
Was this intentionally removed?

Reply:
Yes, this is already included in the `setup`. Not related to the PR, just housekeeping here 😊